### PR TITLE
Add Evennia, the Python MU* Framework, to Awesomo

### PR DESCRIPTION
**Pull request by chainsol**
**Adds [Evennia](https://github.com/evennia/evennia) to Awesomo's Python section**

Evennia is an actively-developed framework for creating and designing multiplayer text-based games - MOO, MUSH, MUD, and everything in-between.

Evennia is BSD-licensed, and instead of using a soft-code system, games are entirely programmed in Python on the filesystem. Evennia provides database abstraction through Django, as well as a built-in webserver, webclient, telnet server, and ssh server, and you can have a bare-bones game running in minutes.

### CODE REVIEW

**Comment by Griatch** [comment]
Looking good - short and concise. Might be good to give the wikipedia link to what MUD means too. How about linking the Evennia logo too, it's here: http://fav.me/dbl6oip

**Comment by chainsol** [comment]
@Griatch Ha, thanks for that link - I linked the one from the GitHub. Will edit that in!

**Comment by lk-geimfari** [comment]
Thank you all, gentlemen's.

**Comment by gtaylor** [comment]
Did this need a separator character above the Evennia entry? It kind of looks like Evennia is under the preceding entry's logo.

File: languages/PYTHON.md [hunk 1]
```diff
@@ -865,4 +865,14 @@ Example:
 
 [![Saleor](http://getsaleor.com/mr-saleor-readme.png)](http://getsaleor.com)
```
Comment (gtaylor) [inline languages/PYTHON.md]: Separator needed here?

Comment (lk-geimfari) [reply languages/PYTHON.md]: No, there is no need. It looks nice. Thank you!

Comment (lk-geimfari) [reply languages/PYTHON.md]: Yeah, actually is needed.

**Comment by lk-geimfari** [comment]
@chainsol Please, look at review by @gtaylor. Thanks!

**Comment by chainsol** [comment]
@gtaylor @lk-geimfari Fixed in #90

File: languages/PYTHON.md (hunk 1 above)
Comment (chainsol) [reply languages/PYTHON.md]: Fixed in #90 , whoops!

**Pull request merged**